### PR TITLE
Update OpenAI image query default model to GPT-4o

### PR DESCRIPTION
**Pull request by andrewfrench**
Update docs and configurations to use GPT-4o to power image queries. Rename OpenAI Image Query Drivers (and descendants) to not specify Vision in their names.

----
📚 Documentation preview 📚: https://griptape--802.org.readthedocs.build//802/

### CODE REVIEW

File: griptape/config/azure_openai_structure_config.py [hunk 1]
```diff
@@ -69,7 +69,7 @@ class AzureOpenAiStructureConfig(StructureConfig):
     )
     image_query_driver: BaseImageQueryDriver = field(
         default=Factory(
-            lambda self: AzureOpenAiVisionImageQueryDriver(
+            lambda self: AzureOpenAiImageQueryDriver(
```
Comment (vachillo) [inline griptape/config/azure_openai_structure_config.py:72]: can you update the model to `gpt-4o` here too since its available?

Comment (andrewfrench) [reply griptape/config/azure_openai_structure_config.py:72]: Sick, will do.

Comment (vachillo) [reply griptape/config/azure_openai_structure_config.py:72]: as well as the prompt driver too

Comment (andrewfrench) [reply griptape/config/azure_openai_structure_config.py:72]: do we need a similar Azure deployment note [here](https://github.com/griptape-ai/griptape/blob/2aed4eb37c5349c720a5d6c3cb7d77590db4c2c8/docs/griptape-framework/drivers/image-query-drivers.md?plain=1#L92) for GPT-4o?

**Review by vachillo — APPROVED** [review]
lgtm, just add a note in the changelog about the azure config updates

File: docs/griptape-framework/drivers/image-query-drivers.md [hunk 2]
```diff
@@ -91,15 +91,15 @@ engine.run("Describe the weather in the image", [image_artifact])
...
 
 ```python
 import os
-from griptape.drivers import AzureOpenAiVisionImageQueryDriver
+from griptape.drivers import AzureOpenAiImageQueryDriver
 from griptape.engines import ImageQueryEngine
 from griptape.loaders import ImageLoader
 
-driver = AzureOpenAiVisionImageQueryDriver(
+driver = AzureOpenAiImageQueryDriver(
```
Comment (vachillo) [inline docs/griptape-framework/drivers/image-query-drivers.md:102]: `AZURE_OPENAI_ENDPOINT_3` and `AZURE_OPENAI_API_KEY_3` can be changed to use `_2`, and the model and azure_deployment can be changed to `gpt-4o`. sorry should have clarified this.

**Pull request merged**